packages/contracts/docs/osx/02-how-to-guides/01-dao/01-best-practices.md

```diff
@@ -2,23 +2,21 @@
 title: Best Practices
 ---
 
-## Advices for Operating your DAO
+## Some Advice When Operating your DAO
 
 ### DOs 👌
 
-- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that it can execute on the DAO.
-- Check every proposal asking to install, update, or uninstall a plugin with utmost care and review
+- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that something can be execute on behalf of the DAO.
+- Check every proposal asking to install, update, or uninstall a plugin with utmost care and review. Installation means granting an external contract permissions, so you want to be extra careful about:
   - the implementation contract
   - the setup contract
   - the helper contracts
   - the permissions
 
 ### DON'Ts ✋
 
-- Incapacitate your DAO by revoking the last `EXECUTE_PERMISSION_ID`.
-  - by uninstalling your last governance plugin.
+- Incapacitate your DAO by revoking the last `EXECUTE_PERMISSION_ID`. This means your DAO will be blocked and any assets you hold may be locked in forever. Revoking `EXECUTION_PERMISSION_ID` is done through:
```

Comment:
```suggestion
- Incapacitate your DAO by revoking the last `EXECUTE_PERMISSION_ID`. This means your DAO will be blocked and any assets you hold may be locked in forever. This can happen through:
```